src/assets/languages/default-en.json

```diff
@@ -1,18 +1,58 @@
 {
   "VENDOR" : "false",
   "SIDEBAR": {
-    "HOME": "Overview",
-    "DASHBOARD": "CI/E2E Dashboard",
-    "DASHBOARD-MDAP": "Enterprise Pipelines",
-    "MDAP":"E2E pipelines for Release branches",
-    "WORKLOADS": "Example Workloads",
-    "ORGANISATION": "MayaData Inc",
-    "MDAPE2E-URL":"https://app.mayaonline.io/mdape2e",
-    "MDAPE2E-TOOLTIP":"Click here to view Enterprise Pipelines",
-    "DASHBOARD-URL": ""
+    "TITLE":"OpenEBS CI Pipelines",
+    "LINKS":{
+      "DEFAULT": "Home",
+      "GROUP-LINKS":{
+        "TITLE":"Storage Engines",
+        "LINK":"{{link_title}}"
+      }
+    },
+    "COPYRIGHT":"Copyright © {{year_num}} MayaData Inc."
   },
-  "WORKLOAD-DASHBOARD": {
-    "TITLE": "Live deployments of stateful applications",
-    "SUBTITLE": "Workloads using Openebs PVs"
+  "ROUTE":{
+    "HOME-ROUTE":{ 
+      "TITLE":"OpenEBS {{branch_title}} Pipelines",
+      "PLATFORM":"{{platform_title}}"
+    },
+    "RECENT":{
+      "TITLE": "Overview",
+      "TABLE":{
+        "HEADER":{
+          "COL-1":"Platform",
```

Comment:
PLATFORM: Platform
Let's use this convention.

Reply:
updated.